mmv1/third_party/terraform/provider/provider_billing_project_test.go

```diff
@@ -293,16 +300,6 @@ resource "google_pubsub_topic" "example-resource-in" {
 // the PubSub API. This allows the second apply step to succeed in a test, if needed.
 func testAccSdkProvider_billing_project_useBillingProject_setupWithApiEnabled(context map[string]interface{}) string {
 	return testAccSdkProvider_billing_project_useBillingProject_setup(context) + acctest.Nprintf(`
-# Needed for post test cleanup
```

Comment:
This has been moved 'up' into `testAccSdkProvider_billing_project_useBillingProject_setup`, so it's still present in `testAccSdkProvider_billing_project_useBillingProject_setupWithApiEnabled`.